### ISSUE TITLE
What format must the target HD for the backup have?

### ISSUE THREAD

**Issue by Golddouble** [bit-team/backintime#1064]
I would like to use an external HD for my backups.

So my question is: 
Can I use a HD that is formated with the microsoft NTFS format? Or must it be the Linux format?

I have this question two times:
1. When I want to backup my home folder
2. When I want to backup the system (for example the /etx folder or the whole root)

**Comment by frontierpsycho** [comment]
I have used NTFS as the file system for the target of backintime, and it generally works quite well. The one issue you might encounter (which I have) is that NTFS supports a narrower character set for file names, which means some files might be perfectly valid on your Linux file system but invalid in NTFS. Moreover, since NTFS is case insenstive, two files with names that only differ in case will clash in the target file system. 

If these aren't issues, you can use NTFS. 

(note that I'm not from the dev team, I'm just a user)

**Comment by emtiu** [comment]
Yes, NTFS is known to be finicky with backintime, because it's really not made to support the *nix idea of files and their properties. For a basic set of user files with tame names, it's probably okay, but I definitely wouldn't rely on it for a full-system backup.

**Comment by Golddouble** [comment]
Thank you both.

**Comment by frontierpsycho** [comment]
Coming back to update my previous comment: if you're using backintime from Linux (which it seems like you will), ie if you are going to use ntfs-3g, backups can sometimes be incredibly slower than they have to be. I switched my backup from an NTFS partition to an ext4 partition on the same external drive and suddenly my backups went from sometimes taking hours or getting stuck and never finishing, to taking minutes.

Moreover, I actually ran into issues with hard links several times, so sometimes a new snapshot would think that files that had already been backed up, were not, and would make fresh copies of them (which I know because they were listed in the changed files).

All in all, I abandoned using backintime (or indeed any backup solution based on rsync) with ntfs-3g.
